x-pack/legacy/plugins/alerting/server/lib/alert_instance.test.ts

```diff
@@ -13,6 +13,23 @@ describe('shouldFire()', () => {
   });
 });
 
+describe('isThrottled', () => {
+  test('defaults to false', () => {
+    const alertInstance = new AlertInstance();
+    expect(alertInstance.isThrottled('1m')).toEqual(false);
+  });
+
+  test('returns true when throttle expiry greater than now', () => {
+    const alertInstance = new AlertInstance({ meta: { lastFired: Date.now() - 59999 } });
```

Comment:
I get all twingey when I see time-specific tests like this.  I think at least allowing more slop - 50000 instead of 59999, for instance.  The test below with 60000 seems safer, but more slop wouldn't hurt.